docs/src/user_docs/stdlib/collections.md

```diff
@@ -0,0 +1,22 @@
+# Collections
+Namespace `std::crypto` contains modules for commonly authenticated data structures.
+
+## Merkle Mountain Range
+Module `std::collections::mmr` contains procedures for manipulating [Merkle Mountain Ranges](https://github.com/opentimestamps/opentimestamps-server/blob/master/doc/merkle-mountain-range.md) data structure which can be used as an append-only log.
+
+| Procedure | Description |
+| ----------- | ------------- |
+| get | Loads the leaf at the absolute `pos` in the MMR onto the stack.<br /><br />This MMR implementation supports only u32 positions.<br /><br />Inputs:<br />- Operand stack: [pos, mmr_ptr, ...]<br /><br />Output:<br />- Operand stack: [N, ...]<br /><br />Where `N` is the leaf loaded from the MMR located starting at memory address `mmr_ptr`. |
+| add | Adds a new leaf to the MMR.<br /><br />This will update the MMR peaks in the VM's memory and the advice provider with any merged nodes.<br /><br />Inputs:<br />- Operand stack: [N, mmr_ptr, ...]<br /><br />Outputs:<br />- Operand stack: [...]<br /><br />Where `N` is the leaf added to the MMR located starting at memory address `mmr_ptr`. |
```

Comment:
nit: is this sentence correct "located starting" or should it be "located at memory address"? O